codecov.yml

```diff
@@ -2,3 +2,7 @@ coverage:
   precision: 2
   round: down
   range: "60...80"
+status:
+  project:
+    default:
+      target: 80
```

Comment:
Incorrect indentation. the `status` key should be aligned with `range`.

Reply:
Fixed in eeaf2c3